.gitmodules

```diff
@@ -6,7 +6,4 @@
 	url = https://github.com/OpenZeppelin/openzeppelin-contracts
 [submodule "lib/forge-std"]
 	path = lib/forge-std
-	url = https://github.com/brockelmore/forge-std
-[submodule "lib/juice-contracts-v1"]
-	path = lib/juice-contracts-v1
-	url = https://github.com/jbx-protocol/juice-contracts-v1
+	url = https://github.com/brockelmore/forge-std
```

Comment:
The full directory shouldn't be included in lib/ (only the symlink), maybe forge remove/install would fix it?

Reply:
+ OZ and Jbx v2 shouldn't be here, since the remapping point to the yarn'd dir, not lib (use forge remove;)